CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.18.0...master
 
+### Fixed
+- Tolerate dirty form warnings in `BackupConfiguration`. Fix [JPERF-967].
```

Comment:
It needs the 'why' explanation in commit body

Reply:
Done